.travis.yml

```diff
@@ -17,12 +17,12 @@ install:
   - ./mvnw -T 1.5C clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q;
 
 before_install:
-  - if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
+  - if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
       openssl aes-256-cbc -K $encrypted_5bb5dbfd81fd_key -iv $encrypted_5bb5dbfd81fd_iv -in travis.tar.gz.enc -out travis.tar.gz -d;
       tar -xzf travis.tar.gz;
       export GOOGLE_APPLICATION_CREDENTIALS=$TRAVIS_BUILD_DIR/travis/admin.json;
       export GOOGLE_CLOUD_PROJECT=cloud-spanner-hibernate-ci;
-      export INTEGRATION_TEST_FLAGS="-Dit.hibernate=true";
+      export INTEGRATION_TEST_FLAGS="-Dit.hibernate=true -Dhibernate.show_sql=false";
```

Comment:
If a test class doesn't print anything for 10 minutes (like the very-long HQLTest class) then it will get killed!

Reply:
Is most of the logging really coming from the actual SQL statements being printed out?

Reply:
Yeah so in the log file I linked, you can see all the statements printed. All of those SQL statements are suppressed with the flag. https://api.travis-ci.org/v3/job/515405979/log.txt

You can see at the end of the log file, our build failed because "The job exceeded the maximum log length, and has been terminated." This PR attempts to solve the problem by suppressing log output; however, maybe this is suppressing too much output...

So far we've identified the following ways our build could fail:
- If no log output is produced in 10 minutes, the build fails
- If the build takes more than 50 minutes, it fails
- If too much log output is produced, the build fails

Reply:
I see a lot of log output from the DEBUG level. Can we just change the log level to INFO?
I think keeping the sql strings in the logs is useful though, unless it is the primary cause of the log getting too large, and not the DEBUG level.